tst/testspecial/backtrace.g.out

```diff
@@ -128,47 +163,156 @@ for i in true do
 od; at *stdin*:27 called from
 <function "f">( <arguments> )
  called from read-eval loop at *errin*:1
+brk> WhereWithVars();
+for i in true do
+    return 1;
+od; at *stdin*:27
+  arguments: <none>
+
```

Comment:
Can we get rid of the empty line here when there are no arguments?

Otherwise: much better :)